specification/schemas/Hook.json

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "object",
+  "required": [
+    "resource_type",
+    "resource_action",
+    "predicate",
+    "action"
```

Comment:
I just realized, a Hook should of course be a proper resource, with an `id`, `type`, `attributes` and `relationships`. And I think it should actually have a relationship with either a `shops`, `organizations`, `brokers` or `enterprises` resource. Although I can't think of a good name for the relationship off the top of my head.

Reply:
And the organizations, brokers and enterprises do not exits jet.